docker-compose-kafka.yml

```diff
@@ -12,6 +12,7 @@ services:
     container_name: kafka-zookeeper
     environment:
       # corresponds to your docker machine and your producer's broker list
+      # if using a recent version of docker for Mac, set the hostname to host.docker.internal
       - KAFKA_ADVERTISED_HOST_NAME=192.168.99.100
```

Comment:
if you aren't using docker-machine (which is probably most people by now), setting an incorrect IP with kafka will result in screenfuls of love cc @openzipkin/devops-tooling 

```             : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 134 : {zipkin=LEADER_NOT_AVAILABLE}
zipkin                      | 2018-08-19 14:25:21.346  WARN 1 --- [pool-2-thread-1] o.a.k.c.NetworkClient                    : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 135 : {zipkin=LEADER_NOT_AVAILABLE}
zipkin                      | 2018-08-19 14:25:21.454  WARN 1 --- [pool-2-thread-1] o.a.k.c.NetworkClient                    : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 136 : {zipkin=LEADER_NOT_AVAILABLE}
zipkin                      | 2018-08-19 14:25:21.562  WARN 1 --- [pool-2-thread-1] o.a.k.c.NetworkClient                    : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 137 : {zipkin=LEADER_NOT_AVAILABLE}
zipkin                      | 2018-08-19 14:25:21.669  WARN 1 --- [pool-2-thread-1] o.a.k.c.NetworkClient                    : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 138 : {zipkin=LEADER_NOT_AVAILABLE}
zipkin                      | 2018-08-19 14:25:21.779  WARN 1 --- [pool-2-thread-1] o.a.k.c.NetworkClient                    : [Consumer clientId=consumer-1, groupId=zipkin] Error while fetching metadata with correlation id 139 : {zipkin=LEADER_NOT_AVAILABLE}
```